.image.env

```diff
@@ -1,8 +1,8 @@
 # Generated file, do not modify.  This file is generated from a text file containing a list of images. The
 # most recent tag is interpolated from the source repository and used to generate a fully qualified image
 # name.
-MINIO_TAG='RELEASE.2023-11-11T08-14-41Z'
-RQLITE_TAG='7.21.4'
-DEX_TAG='v2.37.0'
-SCHEMAHERO_TAG='0.16.0'
+MINIO_TAG='0.20231101.183725'
+RQLITE_TAG='8.0.1'
```

Comment:
anything we need to consider when going from rqlite 7 to 8?

Reply:
could be, i'll downgrade for now even though tests passed.